### DISCUSSION TITLE
Wrong Run Test!!!!!!!

### DISCUSSION THREAD

**Discussion by Moqaddasi** [codewars/codewars.com#2869] [Q&A]
in this challenge
https://www.codewars.com/kata/52449b062fb80683ec000024/train/javascript
I  solve the challenge with this syntax
```
function generateHashtag (str) {
   str = str.charAt(0).toUpperCase() + str.slice(1);
  const value = []
 value.push(str)
  value.unshift ('#')
 return(str.length !=0 && value.join("").length < 140?value.join("").split(" ").join(""):false)
}
```
But the test result said 
Test Results:
Tests
test
Expected a Hashtag (#) at the beginning.: expected '#DoWehaveAHashtag' to equal '#DoWeHaveAHashtag'
Completed in 2ms
Completed in 2ms
I don't know what is wrong with the answer !!!

**Comment by hobovsky** [comment]
Github Discussions are meant to be used for discussions related to Codewars platform and its functions.  
Questions related to specific kata should be asked in [kata discourse](https://www.codewars.com/kata/52449b062fb80683ec000024/discuss/javascript) section.
